app/domain/errors.rb

```diff
@@ -401,6 +401,13 @@ module AuthnJwt
               "   with field in the decoded token that contains the identity",
         code: "CONJ00075E"
       )
+
+      JwtTokenClaimIsMissing = ::Util::TrackableErrorClass.new(
+        msg: "Claim '{0-attribute-name}' is missing from JWT token. " \
+             "Verify that you configured the host with permitted restrictions. " \
+             "In case of Compute Engine token, verify that you requested the token using 'format=full'",
+        code: "CONJ00076E"
```

Comment:
The error code is already in use by `MissingSecretValue` error.

Reply:
There are many duplicated in this file . I think that ordering it should be out of scope of this pr

Reply:
I'm agree ordering is out of scope for this PR
However I think that choosing a unique code for new error is a part of the PR

Reply:
Changed all error codes of jwt exceptions